.github/workflows/deploy-program.yml

```
@@ -92,63 +92,94 @@ jobs:
+      # in a way that round-trips through `solana-verify verify-from-repo`.
+      - name: Install Solana Verify
+        if: github.event.inputs.dry_run == 'false' && inputs.cluster == 'mainnet-beta' && needs.check_tag.outputs.type == 'ref'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libudev-dev pkg-config
+          cargo install solana-verify --locked --force --version "${SOLANA_VERIFY_VERSION}"
+
       - name: Install cargo-release
         uses: metaplex-foundation/actions/install-cargo-release@v1
-        if: github.event.inputs.publish_crate == 'true'
+        if: github.event.inputs.publish_crate == 'true' && github.event.inputs.dry_run == 'false'
```

actionlint (1.7.12)

[error] property "publish_crate" is not defined in object type {cluster: string; dry_run: string; git_ref: string; program: string}

(expression)